scripts/generate_synthetic_pdp_datasets.py

```diff
@@ -19,14 +19,15 @@ def main():
     FAKER.add_provider(pdp.raw_cohort.Provider)
     FAKER.add_provider(pdp.raw_course.Provider)
 
+    # institution_id must be the same for all records.
+    institution_id = FAKER.numerify("#####!")
     cohort_records = [
-        FAKER.raw_cohort_record(normalize_col_names=args.normalize_col_names)
+        FAKER.raw_cohort_record(normalize_col_names=args.normalize_col_names, institution_id=institution_id)
         for _ in range(args.num_students)
     ]
     course_records = [
         FAKER.raw_course_record(
-            cohort_record, normalize_col_names=args.normalize_col_names
-        )
+            cohort_record, normalize_col_names=args.normalize_col_names)
```

Comment:
Just curious, are you using an autoformatter on this code? This parens _should_ be dangling according to ruff 🤷‍♂️